tools/chpl-language-server/src/symbol_signature.py

```diff
@@ -329,3 +345,113 @@ def _fncall_to_string(call: chapel.FnCall) -> List[Component]:
     comps.append(_wrap_str(closebr))
 
     return comps
+
+
+def _opcall_to_string(call: chapel.OpCall) -> List[Component]:
+    """
+    Convert a call to a string
+    """
+
+    def op_to_string(op: str) -> str:
+        special = {"#": "#", ":": ": "}
+        return special.get(op, f" {op} ")
+
+    comps = []
+    if call.is_unary_op():
+        comps.append(_wrap_str(call.op()))
+        comps.extend(_node_to_string(call.actual(0)))
+    else:
+        comps.extend(_node_to_string(call.actual(0)))
+        comps.append(_wrap_str(op_to_string(call.op())))
+        comps.extend(_node_to_string(call.actual(1)))
+    return comps
```

Comment:
You're gonna have a bad time unless you check paren locations, since the AST does not include those. A test case might be: `1 + (1-2)`, or heck, `(1+2)*3`.

Reply:
😞 yeah, I don't know if we keep enough paren location information to inform this? But if we do I will add that support